docs/interaction/rpc.md

```diff
@@ -610,94 +610,243 @@ http post https://rpc.testnet.nearprotocol.com/ jsonrpc=2.0 method=query params:
 
 ## Block
 
-`block(block_height: u64 | block_hash: string)`
+`block({ block_id: BlockId })`
 
-Queries network and returns block for given height. If there was re-org, this may differ.
+`block({ finality: "final" | "near-final" | "optimistic"})`
+
+Learn more about the common identifiers [here](../api/rpc-params).
```

Comment:
prefer absolute links in case these docs move around the filesystem over time.  always start with `/docs/...`